### PR TITLE
Add remove

### PR DESCRIPTION
**Pull request by mmsesay**
In this PR the following was done:

- Functionality to add, update and remove items to  and from the array
- Save and update the data in the local storage
- Functionality to clear all completed tasks
- functionality to refresh the entire list of array
- feature to hide the edit input field on a single click

### CODE REVIEW

File: src/js/data.js [hunk 1]
```diff
@@ -0,0 +1,166 @@
+/* eslint-disable */
```
Comment (carlos-ssh) [inline src/js/data.js]: - Kindly enable your linters and work on your syntax issues.

File: src/js/dragging.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
+/* eslint-disable */
```
Comment (carlos-ssh) [inline src/js/dragging.js]: - Kindly enable your linter and fix your syntax issues.

File: src/js/dragging.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+    nextItemIndex = +nextElement.id;
+    if (nextItemIndex) {
+      const object = this.todoTasks[dragItemIndex];
+
+      this.todoTasks.splice(dragItemIndex, 1);
+      this.todoTasks.splice(nextItemIndex, 0, object);
+    }
+    localStorage.setItem('Tasks', JSON.stringify(this.todoTasks));
+    // this.todoLists.insertBefore(dragItem, nextElement);
```
Comment (carlos-ssh) [inline src/js/dragging.js]: [OPTIONAL]
Comments aren't required if your code explains by itself what it do.

File: README.md [hunk 4]
```diff
@@ -5,6 +5,8 @@ List Structure                                    |
 :------------------------------------------------:|
 ![](./assets/todo-v1.png)                        |
 
+## Online version
+[demo](https://mmsesay.github.io/todo-list/)
 
 ## Built With
```
Comment (carlos-ssh) [inline README.md:12]: - Provide instructions related on how to have a copy locally; how to download, install it, if it's required extra installations.
_This kind of information it's helpful for non-technical people._

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @mmsesay , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/js/data.js (hunk 1 above)
Comment (mmsesay) [reply src/js/data.js]: Thanks for the guide. I have removed it an fix my syntax issues

File: src/js/dragging.js (hunk 3 above)
Comment (mmsesay) [reply src/js/dragging.js]: Okay, noted.

File: README.md (hunk 4 above)
Comment (mmsesay) [reply README.md:12]: Thanks also for this. I have included the instructions in the readme

File: src/js/dragging.js (hunk 2 above)
Comment (mmsesay) [reply src/js/dragging.js]: Thanks for picking this up. I have removed it and fixed all my syntax errors

**Pull request merged**